sdk/monitor/monitor-query/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.2.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.2.0-beta.3 (2023-11-20)
 
 ### Other Changes
 
+- The Monitor Query service has done some internal changes such as changing the type of `interval` from `TimeSpan` to `String` in metrics. These changes are under the generated layer and hidden outside the SDK. A detailed list of changes could be found at [PR #27825](https://github.com/Azure/azure-sdk-for-js/pull/27825/).
```

Comment:
is this even observable to the consumer? If not, maybe it's just better to say we've updated to use the latest service API version and leave it at that?

Reply:
Done